apis/v1beta1/validation/httproute.go

```diff
@@ -334,6 +341,22 @@ func validateHTTPHeaderModifier(filter gatewayv1b1.HTTPHeaderFilter, path *field
 	return errs
 }
 
+func validateHTTPRouteTimeouts(timeouts *gatewayv1b1.HTTPRouteTimeouts, path *field.Path) field.ErrorList {
+	var errs field.ErrorList
+	errorString := fmt.Sprintf("timeout value must be greater than or equal to %d ms", TimeoutMinValue)
+	if timeouts.BackendRequest != nil {
+		if int(timeouts.BackendRequest.Duration.Milliseconds()) >= TimeoutMinValue {
+			errs = append(errs, field.Invalid(path.Child("backendRequest"), timeouts.BackendRequest.Duration, errorString))
+		}
+	}
+	if timeouts.Request != nil {
+		if int(timeouts.Request.Duration.Milliseconds()) >= TimeoutMinValue {
```

Comment:
```suggestion
		if int(timeouts.Request.Duration.Milliseconds()) < TimeoutMinValue {
```

Reply:
addressed in 05f32daa4ee6b188d6f8d07c3777d8ececdb786d